ga4gh/datamodel/variants.py

```diff
@@ -592,16 +592,19 @@ class HtslibVariantAnnotationSet(HtslibVariantSet):
     Class representing a single variant annotation set backed by a directory of
     indexed VCF or BCF files.
     """
-    compoundIdClass = datamodel.VariantAnnotationSetCompoundId
```

Comment:
This is where it should be. It's a class variable, so should be assigned at the class level.